backend/danswer/connectors/confluence/connector.py

```diff
@@ -108,7 +107,7 @@ def __init__(
             )
             self.cql_label_filter = f" and label not in ({comma_separated_labels})"
 
-        self.timezone: timezone = timezone(offset=timedelta(hours=timezone_offset))
+        self.timezone_offset = timezone_offset or CONFLUENCE_TIMEZONE_OFFSET
```

Comment:
The argument `timezone_offset` is `timezone_offset: float = CONFLUENCE_TIMEZONE_OFFSET` - so we can probably remove this

Reply:
true!